src/main/webapp/index.html

```diff
@@ -86,16 +94,20 @@ <h1>Census Viz</h1>
       </div>
       <div id="map-title"></div>
       <div id="more-info"></div>
+      <div id="colors">
+        <label for="setColor">Change the color of the visualization</label>
+        <input type="color" id="setColor" name="setColor" onchange="changeColor(this)">
```

Comment:
I think you'd changed the id already but it's not here anymore for some reason

Reply:
I did revert back to a previous commit because of a bug at some point. Either way, it should be working now. Let me know if there are any other issues.